app/script/conversation/ConversationRepository.js

```diff
@@ -1267,14 +1267,32 @@ z.conversation.ConversationRepository = class ConversationRepository {
    * @returns {Promise} Resolves when conversation was renamed
    */
   renameConversation(conversation_et, name) {
-    return this.conversation_service.updateConversationProperties(conversation_et.id, {name}).then(response => {
+    return this.conversation_service.updateConversationName(conversation_et.id, name).then(response => {
       if (response) {
         amplify.publish(z.event.WebApp.EVENT.INJECT, response, z.event.EventRepository.SOURCE.BACKEND_RESPONSE);
         return response;
       }
     });
   }
 
+  /**
+   * Set the global message timer
+   *
+   * @param {Conversation} conversationEntity - Conversation to update
+   * @param {number} messageTimer - New message timer value
+   * @returns {Promise} Resolves when conversation was updated on server side
+   */
+  updateConversationMessageTimer(conversationEntity, messageTimer) {
+    return this.conversation_service
+      .updateConversationMessageTimer(conversationEntity.id, messageTimer)
+      .then(response => {
+        if (response) {
+          amplify.publish(z.event.WebApp.EVENT.INJECT, response, z.event.EventRepository.SOURCE.BACKEND_RESPONSE);
+          return response;
```

Comment:
One last question: Does the backend response look like the usual backend events or do we have to wrap this?

Reply:
Good question, I have to admit I haven't look at it closely.
Since I am working on server's events on another branch I think I'll go an merge this PR and work on this on the other branch